cmake/assets/run_nodered_template.sh.in

```diff
@@ -1,3 +1,17 @@
-cd @EVEREST_DOCKER_DIR@
-docker build -t everest-nodered .
-docker run --rm --network host --name everest_nodered --mount type=bind,source=@FLOW_FILE@,target=/data/flows.json everest-nodered
+# In the following a volume is created to contain the nodered config, this is done to
+# allow starting the nodered container from inside a devcontainer
+
+# Create docker volume to contain nodered config
+docker volume create nodered-config-volume
+
+# Create temporarily container to copy nodered config into the created volume
+docker create -v nodered-config-volume:/data --name nodered-config-container debian:12-slim true
+
+# Copy nodered config to the created volume with the temporarily created container
+docker cp @FLOW_FILE@ nodered-config-container:/data/flows.json
+
+# Remove temporarily container
+docker rm nodered-config-container
+
+# Start nodered container with the volume mounted to /data
+docker run --rm --network host --name everest_nodered --mount type=volume,source=nodered-config-volume,target=/data ghcr.io/everest/everest-dev-environment/nodered:v0.7.0
```

Comment:
This doesn't work on my end even if I change the not-existing v0.7.0 to docker-images-v0.1.0:

> Unable to find image 'ghcr.io/everest/everest-dev-environment/nodered:docker-images-v0.1.0' locally
docker: Error response from daemon: Head "https://ghcr.io/v2/everest/everest-dev-environment/nodered/manifests/docker-images-v0.1.0": unauthorized.

Reply:
This should be resolved now, it was a wrong setting in the github package settings (visibility), additional I fixed the tag